Docs/reference/content/what_is_new.md

```diff
@@ -13,6 +13,18 @@ title = "What's New"
 Some of the changes in 2.9.0 include:
 
 * Retryable writes are enabled by default
+* Retryable reads support (enabled by default)
```

Comment:
I tried to make this section more concise than our release notes, per previous style